src/pretalx/settings.py

```diff
@@ -206,6 +210,10 @@
     },
 }
 
+if DEBUG:
```

Comment:
Please remove this frontend facing debugging code before the merge, too, including the accompanying CSS styles.